be/src/storage/schema_change.cpp

```diff
@@ -659,52 +652,24 @@ bool ChunkSorter::sort(ChunkPtr& chunk, TabletSharedPtr new_tablet) {
 
 ChunkAllocator::ChunkAllocator(const TabletSchema& tablet_schema, size_t memory_limitation)
         : _tablet_schema(tablet_schema), _memory_limitation(memory_limitation) {
-    _row_len = tablet_schema.row_size();
+    _row_len = tablet_schema.estimate_row_size(8);
```

Comment:
what is the basis of this figure?

Reply:
Before the first chunk is readed, for the `Varchar` type, we can't get the actual size, so we can only conservatively estimate a value.

Reply:
for variable length type

Reply:
Then later, row_len will be adjusted according to the Chunk that has been read

Reply:
I will add some comment

Reply:
> Then later, row_len will be adjusted according to the Chunk that has been read

oh, i know, i have read this related issue.
ok, you can add them, because the code reader maybe cannot comprehend because of the lack of this point as this issue said.

Reply:
Have already add comment